scripts/functional-tests.sh

```diff
@@ -151,7 +151,7 @@ ilab --version
 ilab system info
 
 # pipe 3 carriage returns to ilab config init to get past the prompts
-echo -e "\n\n\n" | ilab init
+echo -e "\n\n\n\n" | ilab init
```

Comment:
(Observation) could as well just `--non-interactive` and never deal with the problem anymore. (Though maybe we should still have a test case for the interactive mode - but more elaborate than just typing Enters.)

Reply:
I changed it to y\n\n\n\n to account for all cases